cpp/arcticdb/pipeline/frame_utils.hpp

```diff
@@ -69,6 +69,22 @@ arcticdb::proto::descriptors::TimeSeriesDescriptor descriptor_from_frame(
     size_t existing_rows,
     std::optional<entity::AtomKey>&& prev_key = {});
 
+template <typename RawType>
+RawType* flatten_tensor(
+        std::optional<ChunkedBuffer>& flattened_buffer,
```

Comment:
As this is effectively a return value, I think it would be clearer to a return a struct or `std::pair`